### PR TITLE
assessment page polish -- UI Polish

### PR DESCRIPTION
**Pull request by rachelchenrc**
Update the visual design for the upfront assessment.

Screen Shot 2020-01-17 at 10 46 24 AM

[Trello card](https://trello.com/c/VlklzvPZ/73-placeholder-upfront-assessment-january-polish)

[LIve Env](https://nj-career-network-dev4.firebaseapp.com/assessment)

### CODE REVIEW

File: src/theme.js [hunk 1]
```diff
@@ -46,13 +47,25 @@ const theme = createMuiTheme({
...
         boxShadow: 'none',
       },
     },
+    MuiStepIcon: {
+      root: {
+        '&$completed': {
+          color: SecondaryMainColor,
+        },
+      },
+      completed: {},
```
Comment (rossdakin) [inline src/theme.js]: Do we need this?

File: components/BackgroundHeader.js [hunk 2]
```diff
@@ -4,33 +4,35 @@ import PropTypes from 'prop-types';
...
 }));
 
 function BackgroundHeader(props) {
-  const { children } = props;
+  const { children, className } = props;
   const classes = useStyles();
 
   return (
-    <Box className={classes.header} display="flex" alignItems="flex-start">
+    <Box className={`${classes.default} ${className}`} display="flex" alignItems="flex-start">
```
[the remark this answers is not in the document]
Comment (rachelchenrc) [reply components/BackgroundHeader.js]: changed to clsx

File: src/theme.js (hunk 1 above)
Comment (rachelchenrc) [reply src/theme.js]: We want to use green for the completed state. As discussed in [trello ](https://trello.com/c/VlklzvPZ/73-placeholder-upfront-assessment-january-polish).

Comment (rossdakin) [reply src/theme.js]: Got it -- please remove unnecessary line `completed: {},`

Comment (rachelchenrc) [reply src/theme.js]: removed

**Pull request merged**